shared/src/common/interfaces/instances/appointment.instance.ts

```diff
@@ -0,0 +1,10 @@
+import { AppointmentType } from '~/common/enums';
+
+export interface IAppointment {
+  date: Date;
+  type: AppointmentType;
+  cost: number;
+  subject: string;
+  createdAt: Date;
+  updatedAt: Date;
+}
```

Comment:
```suggestion
export interface IAppointment {
  date: Date;
  type: AppointmentType;
  cost: number;
  subject: string;
  createdAt: Date;
  updatedAt: Date;
}
```
Let's do the export only at the bottom of the file
For Interfaces and Enums you should do it this way:

`export type { IAppointment }`